modules/odo-key-features.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/understanding-odo.adoc
+
+:_content-type: CONCEPT
+[id="odo-features_{context}"]
+= odo key features
+
+`{odo-title}` is designed to be simple and easy to use, with the following functionality:
```

Comment:
In most cases, throughout this PR, would it make sense to replace `{odo-title}` with `odo`?

Reply:
There are 71 occurrences of `{odo-title}` and 41 occurrences of `odo` in the docs. I think it would be more appropriate to address this problem in its own JIRA.

Reply:
The whole point of odo is to be an easier, more user-friendy, or any of those phases,  interface to  Kubernetes.

I am not sure how we can promote this raison d'etre without using some of those phrases...

I will change it to "simpler and easier to use interface to Kubernetes than oc" - if that isn't true, I'm not sure what to say...